app/src/main/java/com/jerboa/ui/components/post/PostListing.kt

```diff
@@ -419,17 +429,17 @@ fun PostBody(
     }
 }
 
-@Preview
-@Composable
-fun PreviewStoryTitleAndMetadata() {
-    PostBody(
-        postView = samplePostView,
-        onPostLinkClick = {},
-        fullBody = false,
-        expandedImage = false,
-        account = null,
-    )
-}
+//@Preview
```

Comment:
Please restore this and the other previews

Reply:
all of the previews should be restored.